tests/sniffTest

```diff
@@ -68,15 +68,13 @@ if [[ $rc != 0 || -s err ]]; then
 	echo Output: $(cat out)
 	exit 1
 fi
-$RUNC kill c1
-rm -rf /run/opencontainer/containers/c1
```

Comment:
Ok.. refreshed the PR to have it sleep/wait till the container is in the destroyed state.  Also cleaned up the output a bit by removing the trace output.. and fixed the alignment of the -e vs +e option.